src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_enkf_update.F90

```diff
@@ -1955,7 +1955,7 @@ end subroutine output_smapL4SMaup
   ! **********************************************************************
 
   subroutine write_smapL4SMaup( option, date_time, work_path, exp_id, N_ens,    &
-       N_catl, N_catf, N_obsl, tile_coord_f, tile_grid_g, N_catl_vec, low_ind,  &
+       N_catl, N_catf, N_obsl, tile_coord_f, pert_grid_g, N_catl_vec, low_ind,  &
```

Comment:
This subroutines is custom-designed for the SMAP L4_SM product and works **only** if the model is on the EASEv[*] M09 tile space.  It maps data to/from the M09 grid for the purpose of HISTORY-like output.  That is, here we really should use "tile_grid_g", not "pert_grid_g", I think, and make sure that "tile_grid_g" is passed down all the way from the top.   Since thus far we used pert_grid=tile_grid when the model was running in the EASE tile space, this change will not impact the calculations.  But it would be cleaner to use "tile_grid_g" here.  Does this make sense?

Reply:
>  all perturbations and assimilations are on pert_grid

Yes.  But this particular subroutine (`write_smapL4SMaup`) does output on the model grid for the specific case of L4_SM on the M09 EASE grid.  It's an odd case.